src/templates/community-champion-profile.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import Breadcrumb from '~components/dev-hub/breadcrumb';
+import Layout from '~components/dev-hub/layout';
+import { fontSize, screenSize, size } from '~components/dev-hub/theme';
+
+const BREADCRUMB_BOTTOM_MARGIN = '40px';
+
+const ContentContainer = styled('div')`
+    padding: ${size.default} ${size.xxlarge};
+    @media ${screenSize.upToLarge} {
+        padding: ${size.mediumLarge} ${size.default};
+    }
+`;
+
+const StyledBreadcrumb = styled(Breadcrumb)`
+    line-height: 1;
+    margin-bottom: ${BREADCRUMB_BOTTOM_MARGIN};
+    > a {
+        font-size: ${fontSize.xsmall};
+    }
+    @media ${screenSize.upToLarge} {
+        margin-bottom: ${size.large};
+    }
+`;
+
+const CommunityChampionProfile = props => {
+    const {
+        pageContext: { champion, slug },
+    } = props;
+    const { firstName, middleName, lastName } = champion;
+    const fullName = [firstName, middleName, lastName].join(' ');
+    const championProfileBreadcrumbs = [
+        { label: 'Home', target: '/' },
+        {
+            label: 'MongoDB Community Champions',
+            target: '/community-champions',
+        },
+        {
+            label: `${fullName}`,
+            target: slug,
+        },
+    ];
```

Comment:
Nice! Perhaps one slight optimization we can even do is split this up so elements 0 and 1 live in a constant outside of the component and we just `push` 

```
        {
            label: `${fullName}`,
            target: slug,
        },
```

Onto the end. So altogether it might be something like:

```
const CHAMPION_BREADCRUMB_PREFIX = [
        { label: 'Home', target: '/' },
        {
            label: 'MongoDB Community Champions',
            target: '/community-champions',
        },
];

const CommunityChampionProfile = props => {
...
const championProfileBreadcrumbs = championProfileBreadcrumbs.concat([{
            label: `${fullName}`,
            target: slug,
        }])
}
```

Honestly though, this might not even be worth doing so lmk what do you think?

Reply:
hmmm im quite indifferent on this...i'll add it in!

Reply:
Cool, I'm happy either way but just wanted to call it out

Reply:
TY!